### PR TITLE
chore: update test-ssr script

### PR DESCRIPTION
**Pull request by layershifter**
This PR:
- Updates `test-ssr` script to escape the glob
  _Note: some shells evaluate globs, https://github.com/nodejs/node/issues/15529_
- Removes `test-ssr` script from `@fluentui/react-components`, `@fluentui/tokens`, `@fluentui/react-theme`, `@fluentui/react-migration-v8-v9`, `@fluentui/react-conformance-griffel`; are there are no stories to test for SSR
- Updates the generator to emit proper script

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount_base.html">101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount.html">112</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Avatar-mount_base.html">711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Avatar-mount.html">711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Button-mount_base.html">391</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Button-mount.html">390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Field-mount_base.html">1238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Field-mount.html">1232</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProvider-mount_base.html">772</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProvider-mount.html">758</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount_base.html">101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount.html">112</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">94</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">95</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">96</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">95</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/InfoButton-mount_base.html">22</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/InfoButton-mount.html">17</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/MakeStyles-mount_base.html">1233</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/MakeStyles-mount.html">1239</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Persona-mount_base.html">1972</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/Persona-mount.html">1892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/SpinButton-mount_base.html">1492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27916/perf-test-react-components/SpinButton-mount.html">1454</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by Hotell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                   |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                          |          `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>          |          `560 B`<br />`352 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>            |    `3.194 kB`<br />`1.315 kB` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> |  `88.201 kB`<br />`26.692 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                             |   `23.512 kB`<br />`7.197 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                               |   `24.416 kB`<br />`7.506 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                             |   `32.094 kB`<br />`8.367 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                          |   `36.724 kB`<br />`9.458 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                          |  `43.873 kB`<br />`10.939 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                  |  `41.411 kB`<br />`10.791 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                |  `49.635 kB`<br />`12.364 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                              |  `54.866 kB`<br />`11.367 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                      |  `88.428 kB`<br />`25.004 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                |  `83.363 kB`<br />`23.556 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                    |    `9.158 kB`<br />`3.844 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                    |   `11.048 kB`<br />`4.538 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                  |    `9.963 kB`<br />`4.192 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>          |   `91.892 kB`<br />`27.42 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                       |   `17.399 kB`<br />`6.298 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                             |   `18.864 kB`<br />`7.004 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                             |   `11.479 kB`<br />`4.573 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                             |   `24.024 kB`<br />`7.682 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                             |   `10.104 kB`<br />`4.185 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                               |   `12.304 kB`<br />`5.061 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                | `130.392 kB`<br />`39.788 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>   | `133.376 kB`<br />`40.307 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                     |   `11.214 kB`<br />`4.271 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                       | `116.676 kB`<br />`35.957 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                          |   `11.649 kB`<br />`4.262 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                          |   `24.008 kB`<br />`8.798 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                        |   `18.033 kB`<br />`6.666 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                       |   `20.882 kB`<br />`6.798 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                              |    `12.492 kB`<br />`4.92 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                            |   `15.624 kB`<br />`5.232 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>              |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                        |    `31.33 kB`<br />`6.764 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                      |   `17.895 kB`<br />`5.162 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                       |  `46.708 kB`<br />`16.376 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                             |          `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a2a60c86b7d987185142dca378ac2a0ec4e265fb'>a2a60c86b7d987185142dca378ac2a0ec4e265fb</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: packages/react-components/react-conformance-griffel/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "test": "jest --passWithNoTests",
     "type-check": "tsc -b tsconfig.json",
     "generate-api": "just-scripts generate-api",
-    "test-ssr": "test-ssr ./stories/**/*.stories.tsx"
+    "test-ssr": "test-ssr \"./stories/**/*.stories.tsx\""
```
[the remark this answers is not in the document]
Comment (layershifter) [reply packages/react-components/react-conformance-griffel/package.json]: Removed it

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by layershifter** [comment]
Build is passing, I have an approval from @microsoft/fluentui-react-build ✅ Force merging as it's a simple change.

**Pull request merged**